dev-client/src/components/tables/DataGridTable.tsx

```diff
@@ -52,16 +56,31 @@ export const DataGridTable = ({rows, headers, ...containerProps}: Props) => {
     <Box {...containerProps}>
       <Row justifyContent="flex-start" variant="tablerow">
         {headers.map((header: (typeof headers)[number], i: number) => (
-          <Text variant="table-header" key={i} flex={1} {...textAlign}>
+          <Text
+            variant="table-header"
+            key={i}
+            flex={1}
+            py="10px"
+            {...textAlign}>
             {header}
           </Text>
         ))}
       </Row>
+      <Divider />
       {rows.map((row: (typeof rows)[number], i: number) => (
-        <Row justifyContent="flex-start" key={i} variant="tablerow">
-          {row.map(displayCol)}
-        </Row>
+        <>
+          <Row justifyContent="flex-start" key={i} variant="tablerow" py="10px">
```

Comment:
The first time I open the soil pit section on the Project screen, I get this error in the console:

` ERROR  Warning: Each child in a list should have a unique "key" prop.`

Might be worth looking into, not sure if this ticket introduced it.

Reply:
Good catch. Should be fixed.